rotkehlchen/globaldb/schema.py

```diff
@@ -230,6 +256,12 @@
 );
 """
 
+# asset_collections allows to set common information for related assets as descrbed in the
```

Comment:
Maybe move `asset_collections` creation before `multiasset_mappings` to have a better "plot" ?